newton/tests/test_implicit_pd.py

```
@@ -0,0 +1,342 @@
+    def test_kamino_cases(self):
+        kp = 200.0
+        kd = 20.0
+        dt = 1.0 / 30.0
+
+        implicit_q, implicit_qd = _run_kamino_pd_case(
+            kp, kd, dt, use_model_pd=True, use_actuator=False, use_actuator_jacobians=False
+        )
+        jacobian_q, jacobian_qd = _run_kamino_pd_case(
+            kp, kd, dt, use_model_pd=True, use_actuator=True, use_actuator_jacobians=True
+        )
+        explicit_q, explicit_qd = _run_kamino_pd_case(
```

Ruff (0.15.21)

[warning] Unpacked variable `explicit_qd` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)